code/modules/mob/mob_movement.dm

```diff
@@ -505,10 +505,19 @@
 	var/turf/above_turf = GET_TURF_ABOVE(current_turf)
 
 	var/ventcrawling_flag = HAS_TRAIT(src, TRAIT_MOVE_VENTCRAWLING) ? ZMOVE_VENTCRAWLING : 0
+
+	if(src.remote_control)
+		current_turf = get_turf(src.remote_control)
+		above_turf = GET_TURF_ABOVE(current_turf)
+		ventcrawling_flag = HAS_TRAIT(src.remote_control, TRAIT_MOVE_VENTCRAWLING) ? ZMOVE_VENTCRAWLING : 0
```

Comment:
This line is useless since we are just going call `return src.remote_control.relaymove(src, UP)` if `remote_control` is set.